src/plugins/pcr_primer_design_for_dna_assembly/src/PCRPrimerDesignForDNAAssemblyOPWidget.h

```diff
@@ -76,15 +77,21 @@ private slots:
      * Show the "Open file" dialog, choose a file with other PCR sequences and set path to the corresponding line edit.
      */
     void sl_loadOtherSequenceInPcr();
+    void sl_onFindTaskFinished();
 
 private:
-    AnnotatedDNAView* annDnaView = nullptr;
+    void showResults();
+    void createResultAnnotations();
+
+    AnnotatedDNAView *annDnaView = nullptr;
 
     QToolButton* smButton = nullptr;
     QSpinBox* sbStartRegion = nullptr;
     QSpinBox* sbEndRegion = nullptr;
     QMetaObject::Connection updateRegionConnection;
 
+    PCRPrimerDesignForDNAAssemblyTask *pcrTask;
```

Comment:
Use the modern way to initialize variables `PCRPrimerDesignForDNAAssemblyTask *pcrTask = nullptr;`instead of initializing the into a Constructor